ccd-definition/CaseEvent/CareSupervision/MultiState.json

```diff
@@ -36,7 +36,8 @@
     "Name": "Update case",
     "PreConditionState(s)": "*",
     "PostConditionState": "*",
-    "SecurityClassification": "Public"
+    "SecurityClassification": "Public",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_CASE_SERVICE_BASE_URL}/callback/generic-update/about-to-submit"
```

Comment:
Just for my understanding ... are you adding this only to events where we want to trigger the new interceptor?